src/functional/program_processors/codegens/gtfn/codegen.py

```diff
@@ -216,7 +225,7 @@ def visit_FencilDefinition(
 
     inline auto ${id} = [](auto... connectivities__){
         return [connectivities__...](auto backend, ${','.join('auto&& ' + p for p in params)}){
-            auto tmp_alloc__ = gtfn::backend::tmp_allocator(backend);
```

Comment:
I'm not sure why I need this change, but I positively need this in order to compile, at least on my local tsa installation

Reply:
Can you post the error.

Reply:
Apparently I can't because it just seems to work now. Feel like I'm going insane.